src/js/serviceWorker/tests/oninstallAndUpdate.test.js

```diff
@@ -1,19 +1,29 @@
 import { onInstall, onUpdate } from '../onInstallAndUpdate';
 
 global.chrome = {
-  windows: {
-    create: jest.fn(),
-  },
+    windows: {
+        create: jest.fn(),
+    },
   storage: {
     sync: {
       set: jest.fn(),
     }
-  }
+  },
+    action: {
```

Comment:
will inheriting the rest of global.chrome instead of overriding the object allow us to not re-mock this?